src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactory.java

```diff
@@ -264,20 +256,21 @@ private Mono<PostgresqlConnection> doCreateConnection(UniformLoadBalancerConnect
                             })
                             .delayUntil(connection -> {
                                 return prepareConnection(connection, client.getByteBufAllocator(), codecs, forReplication);
-                            })
-                            .onErrorResume(throwable -> this.closeWithError(client, throwable));
-                }).onErrorMap(e -> {
-                    if (e instanceof R2dbcException) {
-                        return e;
-                    }
-                    return new R2dbcNonTransientResourceException(String.format("Cannot create control connection using %s", this.connectionFunction), e);
+                            });
+                })
+                .onErrorResume(throwable -> {
+                    System.out.println(host + " not reachable, adding to failed list");
```

Comment:
Should be some logger API instead of `System.out.println`

Reply:
There is no logger API, keeping it `System.out.println` only for now or should I remove it altogether?

Reply:
Please remove